x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -135,6 +135,8 @@ export class TaskRunner<
 
     const path = addSpaceIdToPath('/', spaceId);
 
+    // calc a hash to have stable id across the different calls
+    const requestId = apiKey ? uuidv5(apiKey, uuidv5.DNS) : undefined;
```

Comment:
according to https://en.wikipedia.org/wiki/Universally_unique_identifier#Versions_3_and_5_(namespace_name-based):
> neither the namespace nor name can be determined from the UUID

@pmuellr let me know if you want me to roll this change back. I noticed that I cannot rely on `uuid/v4` since this method is called twice during `task.run` which generates different ids.